File: Profile.UI/src/app/models/education.ts
export class Education {
    id: string = "";
    institutionName: string = "";
    degree: string = "";
    fieldOfStudy: string = "";
    startDate: string = "";
    endDate: string = "";
    description: string = "";

    constructor(id: string, degree: string, institutionName: string, startDate: string, endDate: string, description: string) {
        this.id = id;
        this.degree = degree;
        this.institutionName = institutionName;
        this.startDate = startDate;
        this.endDate = endDate;
        this.description = description;
    }
}

File: Profile.Core/Profile.Domain/Entities/Education.cs
using Profile.Domain.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Profile.Domain.Entities
{
    public class Education : BaseEntity
    {
        public string InstitutionName { get; set; }
        public string Degree { get; set; }
        public string FieldOfStudy { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Description { get; set; }

        public Education(string institutionName, string degree, string fieldOfStudy, DateTime startDate, DateTime endDate, string description)
        {
            this.InstitutionName = institutionName;
            this.Degree = degree;
            this.FieldOfStudy = fieldOfStudy;
            this.StartDate = startDate;
            this.EndDate = endDate;
            this.Description = description;
        }
    }
}
